assets/dev/js/editor/views/add-section/base.js

```diff
@@ -154,66 +154,16 @@ class AddSectionBase extends Marionette.ItemView {
 	 *
 	 * @param {MouseEvent} e - Click event.
 	 *
-	 * @return void
+	 * @return {Container}
 	 */
 	onContainerPresetSelected( e ) {
 		this.closeSelectPresets();
 
-		const selectedPreset = e.currentTarget.dataset.preset;
-		let newContainer;
-
-		switch ( selectedPreset ) {
-			// Single Container without sub Containers.
-			case '100':
-				newContainer = ContainerHelper.createContainer( {}, elementor.getPreviewContainer(), this.options );
-				break;
-
-			// Exceptional preset.
-			case 'c100-c50-50':
-				newContainer = ContainerHelper.createContainer( {
-					flex_direction: ContainerHelper.DIRECTION_ROW,
-					flex_wrap: 'wrap',
-					flex_gap: {
-						unit: 'px',
-						size: 0, // Set the gap to 0 to override the default inherited from `Site Settings`.
-					},
-				}, elementor.getPreviewContainer(), this.options );
-
-				const settings = {
-					width: {
-						unit: '%',
-						size: '50',
-					},
-					width_mobile: {
-						unit: '%',
-						size: '100',
-					},
-				};
-
-				ContainerHelper.createContainer( settings, newContainer, { edit: false } );
-
-				const rightContainer = ContainerHelper.createContainer( {
-					...settings,
-					padding: { size: '' }, // Create the right Container with 0 padding (default is 10px) to fix UI (ED-4900).
-					flex_gap: {
-						unit: 'px',
-						size: 0, // Set the gap to 0 to override the default inherited from `Site Settings`.
-					},
-				}, newContainer, { edit: false } );
-
-				ContainerHelper.createContainers( 2, {}, rightContainer, { edit: false } );
-
-				break;
-
-			// Containers by preset.
-			default:
-				newContainer = ContainerHelper.createContainerFromPreset(
-					selectedPreset,
-					elementor.getPreviewContainer(),
-					this.options
-				);
-				break;
-		}
+		return ContainerHelper.createContainerFromPreset(
+			e.currentTarget.dataset.preset,
+			elementor.getPreviewContainer(),
+			this.options
+		);
```

Comment:
@StyleShit also this should be in your PR.

Reply:
Just open a PR to `feature/container` with all of those changes.